ecs-agent/daemonimages/csidriver/Makefile

```diff
@@ -1,27 +1,39 @@
-CSI_DRIVER_VERSION="v$(shell cat ./container/VERSION)"
-GO111MODULE=on
-GOPATH=$(shell go env GOPATH)
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+.PHONY: all tar image mockgen clean
+
+CSI_DRIVER_VERSION="v$(shell cat CSI_DRIVER_VERSION)"
 BUILD_DATE=$(shell date -u -Iseconds)
+GO_VERSION=$(shell cat ../../../GO_VERSION)
 
-OS?=linux
-ARCH?=amd64
+all: tar
 
-.PHONY: bin/csi-driver
-bin/csi-driver:
-	CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -ldflags "\
+bin/aws-ebs-csi-driver:
+	CGO_ENABLED=0 go build -ldflags "\
 		-X \"github.com/aws/amazon-ecs-agent/ecs-agent/daemonimages/csidriver/version.version=$(CSI_DRIVER_VERSION)\" \
 		-X \"github.com/aws/amazon-ecs-agent/ecs-agent/daemonimages/csidriver/version.buildDate=$(BUILD_DATE)\"" \
-		-o ./bin/csi-driver ./
+		-o bin/aws-ebs-csi-driver .
+
+image:
+	docker build --build-arg GO_VERSION=$(GO_VERSION) -t aws-ebs-csi-driver:make .
```

Comment:
Can we use this image ref - `ebs-csi-driver:latest` which is hardcoded in fargate side?

Reply:
thx for pointing out, fixed